src/layout/mod.rs

```diff
@@ -1196,6 +1196,36 @@ impl<W: LayoutElement> Layout<W> {
         monitor.move_column_to_last();
     }
 
+    pub fn move_column_left_or_output(&mut self, output: &OutputId) -> bool {
```

Comment:
```suggestion
    pub fn move_column_left_or_to_output(&mut self, output: &Output) -> bool {
```
These should accept `&Output` not `&OutputId`.